doc/connectivity/networking/native_sim_setup.rst

```diff
@@ -145,18 +145,27 @@ For TCP test, type:
 
    ./echo-client -t 127.0.0.1
 
-Setting interface name from command line
-****************************************
+Setting interface name and IPv4 parameters from command line
+************************************************************
 
 By default the Ethernet interface name used by native_sim is determined by
 :kconfig:option:`CONFIG_ETH_NATIVE_POSIX_DRV_NAME`, but is also possible
 to set it from the command line using ``--eth-if=<interface_name>``.
+
+The same applies to the IPv4 address, gateway and netmask. The default values
```

Comment:
Also note that this is only true if `CONFIG_NET_CONFIG_SETTINGS` is actually enabled, otherwise nothing would be configured. So if that was moved directly to the driver instead, we could rely on those command-line args regardless of other configs.

Reply:
@rlubos has a point, it might make more sense to place this support to native_sim Ethernet driver.